spec/unit/parser_file_spec.rb

```diff
@@ -3,58 +3,78 @@
 describe GitHubChangelogGenerator::ParserFile do
   describe ".github_changelog_generator" do
     let(:options) { {} }
+    let(:parser) { GitHubChangelogGenerator::ParserFile.new(options, StringIO.new(file)) }
```

Comment:
Minor: if you like you can use described_class.

Reply:
Yes I like it ❤️